src/test/java/org/llorllale/cactoos/matchers/HasValuesMatchingTest.java

```diff
@@ -47,10 +47,10 @@
      */
```

Comment:
why do we need 'PMD.AvoidDuplicateLiterals' here?

Reply:
@ilyakharlamov removed